evalml/pipelines/components/estimators/regressors/rf_regressor.py

```diff
@@ -53,3 +58,49 @@ def __init__(
             component_obj=rf_regressor,
             random_seed=random_seed,
         )
+
+    def get_prediction_intervals(
+        self,
+        X: pd.DataFrame,
+        y: pd.Series = None,
+        coverage: List[float] = None,
+    ) -> Dict[str, pd.Series]:
+        """Find the prediction intervals using the fitted RandomForestRegressor.
+
+        Args:
+            X (pd.DataFrame): Data of shape [n_samples, n_features].
+            y (pd.Series): Target data. Optional.
+            coverage (list[float]): A list of floats between the values 0 and 1 that the upper and lower bounds of the
+                prediction interval should be calculated for.
+
+        Returns:
+            dict: Prediction intervals, keys are in the format {coverage}_lower or {coverage}_upper.
+        """
+        if coverage is None:
```

Comment:
would we expect the implementation of rf and ETS to change in the future? Maybe it would be worth it to standardize the method somewhere and use the same impl in both estimators.